<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="3" failed="0" ignored="0" total="3" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="5303" started-at="2019-01-29T23:53:27Z" finished-at="2019-01-29T23:53:33Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="5303" started-at="2019-01-29T23:53:27Z" finished-at="2019-01-29T23:53:33Z">
      <class name="MobileProgrammingLLC.Tests.APIDemos_SettingWiFi">
        <test-method status="FAIL" signature="initConfigs()[pri:0, instance:MobileProgrammingLLC.Tests.APIDemos_SettingWiFi@639c2c1d]" name="initConfigs" is-config="true" duration-ms="5212" started-at="2019-01-29T23:53:27Z" finished-at="2019-01-29T23:53:33Z">
          <exception class="io.appium.java_client.service.local.InvalidServerInstanceException">
            <message>
              <![CDATA[Invalid server instance exception has occured: There is no installed nodes! Please install node via NPM (https://www.npmjs.com/package/appium#using-node-js) or download and install Appium app (http://appium.io/downloads.html)]]>
            </message>
            <full-stacktrace>
              <![CDATA[io.appium.java_client.service.local.InvalidServerInstanceException: Invalid server instance exception has occured: There is no installed nodes! Please install node via NPM (https://www.npmjs.com/package/appium#using-node-js) or download and install Appium app (http://appium.io/downloads.html)
at io.appium.java_client.service.local.AppiumServiceBuilder.findNodeInCurrentFileSystem(AppiumServiceBuilder.java:156)
at io.appium.java_client.service.local.AppiumServiceBuilder.checkAppiumJS(AppiumServiceBuilder.java:317)
at io.appium.java_client.service.local.AppiumServiceBuilder.createArgs(AppiumServiceBuilder.java:397)
at org.openqa.selenium.remote.service.DriverService$Builder.build(DriverService.java:338)
at io.appium.java_client.service.local.AppiumDriverLocalService.buildService(AppiumDriverLocalService.java:71)
at MobileProgrammingLLC.Resources.Base.startAppiumServer(Base.java:72)
at MobileProgrammingLLC.Resources.Base.createAppiumServerConnection(Base.java:124)
at MobileProgrammingLLC.Tests.APIDemos_SettingWiFi.initConfigs(APIDemos_SettingWiFi.java:36)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:168)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.io.IOException: 
┌──────────────────────────────────────────────────────────────────┐
│                     npm update check failed                      │
│               Try running with sudo or get access                │
│               to the local update config store via               │
│ sudo chown -R $USER:$(id -gn $USER) /Users/love.vashista/.config │
└──────────────────────────────────────────────────────────────────┘
/usr/local/lib/node_modules

at io.appium.java_client.service.local.AppiumServiceBuilder.findNodeInCurrentFileSystem(AppiumServiceBuilder.java:157)
... 33 more
]]>
            </full-stacktrace>
          </exception> <!-- io.appium.java_client.service.local.InvalidServerInstanceException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initConfigs -->
        <test-method status="SKIP" signature="clickOnPreferenceOption()[pri:0, instance:MobileProgrammingLLC.Tests.APIDemos_SettingWiFi@639c2c1d]" name="clickOnPreferenceOption" duration-ms="0" started-at="2019-01-29T23:53:33Z" finished-at="2019-01-29T23:53:33Z">
          <exception class="io.appium.java_client.service.local.InvalidServerInstanceException">
            <message>
              <![CDATA[Invalid server instance exception has occured: There is no installed nodes! Please install node via NPM (https://www.npmjs.com/package/appium#using-node-js) or download and install Appium app (http://appium.io/downloads.html)]]>
            </message>
            <full-stacktrace>
              <![CDATA[io.appium.java_client.service.local.InvalidServerInstanceException: Invalid server instance exception has occured: There is no installed nodes! Please install node via NPM (https://www.npmjs.com/package/appium#using-node-js) or download and install Appium app (http://appium.io/downloads.html)
at io.appium.java_client.service.local.AppiumServiceBuilder.findNodeInCurrentFileSystem(AppiumServiceBuilder.java:156)
at io.appium.java_client.service.local.AppiumServiceBuilder.checkAppiumJS(AppiumServiceBuilder.java:317)
at io.appium.java_client.service.local.AppiumServiceBuilder.createArgs(AppiumServiceBuilder.java:397)
at org.openqa.selenium.remote.service.DriverService$Builder.build(DriverService.java:338)
at io.appium.java_client.service.local.AppiumDriverLocalService.buildService(AppiumDriverLocalService.java:71)
at MobileProgrammingLLC.Resources.Base.startAppiumServer(Base.java:72)
at MobileProgrammingLLC.Resources.Base.createAppiumServerConnection(Base.java:124)
at MobileProgrammingLLC.Tests.APIDemos_SettingWiFi.initConfigs(APIDemos_SettingWiFi.java:36)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:168)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.io.IOException: 
┌──────────────────────────────────────────────────────────────────┐
│                     npm update check failed                      │
│               Try running with sudo or get access                │
│               to the local update config store via               │
│ sudo chown -R $USER:$(id -gn $USER) /Users/love.vashista/.config │
└──────────────────────────────────────────────────────────────────┘
/usr/local/lib/node_modules

at io.appium.java_client.service.local.AppiumServiceBuilder.findNodeInCurrentFileSystem(AppiumServiceBuilder.java:157)
... 33 more
]]>
            </full-stacktrace>
          </exception> <!-- io.appium.java_client.service.local.InvalidServerInstanceException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- clickOnPreferenceOption -->
        <test-method status="SKIP" signature="clickOnPreferenceDependenciesOption()[pri:0, instance:MobileProgrammingLLC.Tests.APIDemos_SettingWiFi@639c2c1d]" name="clickOnPreferenceDependenciesOption" duration-ms="0" started-at="2019-01-29T23:53:33Z" depends-on-methods="MobileProgrammingLLC.Tests.APIDemos_SettingWiFi.clickOnPreferenceOption" finished-at="2019-01-29T23:53:33Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method APIDemos_SettingWiFi.clickOnPreferenceDependenciesOption()[pri:0, instance:MobileProgrammingLLC.Tests.APIDemos_SettingWiFi@639c2c1d] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method APIDemos_SettingWiFi.clickOnPreferenceDependenciesOption()[pri:0, instance:MobileProgrammingLLC.Tests.APIDemos_SettingWiFi@639c2c1d] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:887)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- clickOnPreferenceDependenciesOption -->
        <test-method status="SKIP" signature="settingWifiFunctionality()[pri:0, instance:MobileProgrammingLLC.Tests.APIDemos_SettingWiFi@639c2c1d]" name="settingWifiFunctionality" duration-ms="0" started-at="2019-01-29T23:53:33Z" depends-on-methods="MobileProgrammingLLC.Tests.APIDemos_SettingWiFi.clickOnPreferenceDependenciesOption" finished-at="2019-01-29T23:53:33Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method APIDemos_SettingWiFi.settingWifiFunctionality()[pri:0, instance:MobileProgrammingLLC.Tests.APIDemos_SettingWiFi@639c2c1d] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method APIDemos_SettingWiFi.settingWifiFunctionality()[pri:0, instance:MobileProgrammingLLC.Tests.APIDemos_SettingWiFi@639c2c1d] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:887)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- settingWifiFunctionality -->
        <test-method status="SKIP" signature="tearDown()[pri:0, instance:MobileProgrammingLLC.Tests.APIDemos_SettingWiFi@639c2c1d]" name="tearDown" is-config="true" duration-ms="0" started-at="2019-01-29T23:53:33Z" finished-at="2019-01-29T23:53:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- MobileProgrammingLLC.Tests.APIDemos_SettingWiFi -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
